staging/src/k8s.io/client-go/kubernetes/typed/apps/v1/daemonset.go

```diff
@@ -43,6 +40,7 @@ type DaemonSetsGetter interface {
 type DaemonSetInterface interface {
 	Create(ctx context.Context, daemonSet *v1.DaemonSet, opts metav1.CreateOptions) (*v1.DaemonSet, error)
 	Update(ctx context.Context, daemonSet *v1.DaemonSet, opts metav1.UpdateOptions) (*v1.DaemonSet, error)
+	// Add a +genclient:noStatus comment above the type to avoid generating UpdateStatus().
```

Comment:
I am unclear why these comments are net-new.  Which part of this PR caused them to start being emitted?

Reply:
It’s this change to `generator_for_type.go`: https://github.com/kubernetes/kubernetes/pull/121439/files#diff-ebf0a2af8ee60740c8f5b77c6fad05b9690d9b371e96cae8f4e1f77f8ca09870R364

Before this PR the comments were part of the comments on each `UpdateStatus()` function declaration: https://github.com/kubernetes/kubernetes/pull/121439/files#diff-ebf0a2af8ee60740c8f5b77c6fad05b9690d9b371e96cae8f4e1f77f8ca09870L628